packages/kaizen-tg/_templates/drupal-9-theme/new/.storybook/preview.js

```diff
@@ -9,9 +9,30 @@ window.themeBreakpoints = config;
 
 // Import sprite.
 import svgSpritePath from '../dist/svg/sprite.svg';
-window.svgSpritePath = svgSpritePath;
+window.<%= h.changeCase.lower(name) %>SvgSpritePath = svgSpritePath;
 
 // Extend Twig.js with drupal filters.
 import Twig from 'twig';
 import twigDrupalFilters from 'twig-drupal-filters';
 twigDrupalFilters(Twig);
+
+window.Drupal = { behaviors: {}, locale: {} };
+
+((Drupal, drupalSettings) => {
+  Drupal.attachBehaviors = function (context, settings) {
```

Comment:
should be also `throw error`
like in https://github.com/emulsify-ds/emulsify-drupal/blob/master/.storybook/_drupal.js

Reply:
added